src/DataDictionary/DataDictionaryNode.jsx

```diff
@@ -40,7 +38,7 @@ const LinkTable = ({ links }) => {
       <TableHead>
         <TableRow>
           {fields.map(field =>
-            <TableData first_cr key={field}>{field}</TableData>)}
+            <td className="data-dictionary__table-data data-dictionary__table-data--head" key={field}>{field}</td>)}
```

Comment:
does `first_cr` need to be in this still? also why is the `table-data--head` class applied as well?